src/media_devices/mod.rs

```diff
@@ -89,6 +89,51 @@ impl MediaDeviceInfo {
     }
 }
 
+/// Dictionary used to instruct what sort of tracks to include in the [`MediaStream`] returned by
+/// [`get_user_media`]
```

Comment:
I found this to be the most friendly way of modeling the constraints, but let me know if you think otherwise